RecoTauTag/RecoTau/plugins/PFRecoTauChargedHadronProducer.cc

```diff
@@ -164,6 +168,9 @@ void PFRecoTauChargedHadronProducer::produce(edm::Event& evt, const edm::EventSe
   // loop over our jets
   BOOST_FOREACH( const reco::PFJetRef& pfJet, pfJets ) {
     
+    if(pfJet->pt() - minJetPt_ < 1e-5) continue;
+    if(std::abs(pfJet->eta()) - maxJetAbsEta_ > -1e-5) continue;
```

Comment:
what's the significance of this ``` 1e-5``` ?
Can it be removed and the logic changed to just ```pfJet->pt() < minJetPt_  ``` (and similarly for eta)?